src/fetchPng.js

```diff
@@ -0,0 +1,17 @@
+import { PNG } from 'pngjs';
+import request from 'request';
+
+export default async function fetchPng(url) {
+  return new Promise((resolve, reject) => {
+    request({
+      url,
+    })
+      .on('error', (e) => {
+        reject(e);
+      })
+      .pipe(new PNG())
+      .on('parsed', function handleParsed() {
```

Comment:
It feels awkward and unclear that pngjs uses `this` instead of passing the parsed object as an arg to the handler. /shrug

Although, it looks like this function receives `data` as an arg, which is maybe the bitmap you need? https://www.npmjs.com/package/pngjs#event-parsed

Reply:
It wasn't, I thought so too :(